.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+
+before_install:
+  - composer self-update
+
+install:
+  - composer install --prefer-source --no-interaction --no-progress
```

Comment:
You're right, I didn't see the related PR, do you suggest to add the `--prefer-dist` and something to configure the token too?